web/rootfs/defaults/meet.conf

```diff
@@ -81,12 +81,10 @@ location /etherpad/ {
     proxy_http_version 1.1;
     proxy_set_header Upgrade $http_upgrade;
     proxy_set_header Connection 'upgrade';
-    proxy_set_header Host $host;
     proxy_cache_bypass $http_upgrade;
 
     proxy_pass {{ .Env.ETHERPAD_URL_BASE }}/;
     proxy_set_header X-Forwarded-For $remote_addr;
     proxy_buffering off;
-    proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
```

Comment:
Is removing this necessary?

Reply:
Yes, Removing one of the headers results in a 404 either on the meet server, or the etherpad server.